samples/suit/smp_transfer/boards/nrf54h20dk_nrf54h20_cpuapp_extflash.overlay

```diff
@@ -22,3 +22,24 @@
 		};
 	};
 };
+
+&cpuapp_rx_partitions {
+	/delete-node/ partition@b7000;
+	/delete-node/ cpuapp_slot0_partition;
+	/delete-node/ cpuapp_recovery_partition;
+	cpuapp_recovery_partition: partition@a5000 {
+		reg = <0xa5000 DT_SIZE_K(84)>;
+	};
+
+	cpuapp_slot0_partition: partition@ba000 {
+		reg = <0xba000 DT_SIZE_K(1124)>;
+	};
+};
+
+&cpuapp_rw_partitions {
+	/delete-node/ partition@100000;
+	/delete-node/ dfu_partition;
```

Comment:
I have a feeling that line 40 and 41 deletes the same partition... is that true?